include/zephyr/drivers/sensor/mcux_lpcmp.h

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2024 NXP
```

Comment:
This file looks like it is based off `include/zephyr/drivers/sensor/mcux_acmp.h`, yet copyright have not been retained.

Reply:
added, please check